tools/cutadapt_fixed_length.cwl

```diff
@@ -0,0 +1,31 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: cutadapt-fixed-length
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/cutadapt:3.4'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: 8
+    ramMin: 16000
+
+arguments:
+  - position: 1
+    shellQuote: false
+    valueFrom: >-
+      cutadapt -j 8 --length $(inputs.fixed_length[0]) -o TRIMMED.$(inputs.readFilesIn1.basename)
+  - position: 3
+    shellQuote: false
+    valueFrom: >-
+      && cutadapt -j 8 --length $(inputs.fixed_length[1]) -o TRIMMED.$(inputs.readFilesIn2.basename)
+
+
+inputs:
+  fixed_length: { type: 'int[]', doc: "Shorten reads to a fixed length. Useful for specialized protocols in which the instrument will read beyond useable sequence" }
```

Comment:
I think we should just call these something line:
- `r1_max_len`
- `r2_max_len`

The array for this makes it seem like upper and lower boundaries. It's not very clear what's going on.